t/sanity.t

```diff
@@ -1,6 +1,7 @@
 use lib 't';
 use Test::APIcast::Blackbox 'no_plan';
 
+filters { configuration => 'fixture=echo.json' };
```

Comment:
I understand that the `fixture=something` part is implemented in Test::APIcast 0.10, but where is the `filters { something => something_else } ` defined? Is it just valid for blocks?

Reply:
@davidor that comes from http://search.cpan.org/~ingy/Test-Base-0.88/lib/Test/Base.pod#FILTERS 
That is what powers the Test::Nginx declarative testing blocks.

Reply:
:+1: